spot-vaults/contracts/BillBroker.sol

```diff
@@ -868,23 +880,29 @@ contract BillBroker is
             return MAX_FEE_FACTOR;
         }
 
-        Range memory f1 = fees.usdToPerpSwapFeeFactors;
-        Range memory f2 = fees.perpToUSDSwapFeeFactors;
+        Range memory feeU2P = fees.usdToPerpSwapFeeFactors;
+        Range memory feeP2U = fees.perpToUSDSwapFeeFactors;
+        uint256 feeP2UDelta = feeP2U.upper - feeP2U.lower;
         return
             LineHelpers
                 .computePiecewiseAvgY(
                     Line({
                         x1: arSoftBound.lower,
-                        y1: f1.lower,
+                        y1: feeU2P.lower,
```

Comment:
I would swap (x1,y1) with (x2,y2) here, so that we're always having the line segments go rightward on the x-axis. I think this should be mathematically equivalent.